crates/aptos/src/common/types.rs

```diff
@@ -738,6 +738,40 @@ pub trait ParsePrivateKey {
     }
 }
 
+#[derive(Debug, Default, Parser)]
+pub struct HardwareWalletOptions {
```

Comment:
is this type for any hardware wallet or only ledger? if it is the latest, should probably name it `LedgerWalletOption`

Reply:
Yeah, I am actually planning it for Hardware overall, I believe all HD wallet should have derivation path